content/en/stateless_apps.md

```diff
@@ -6,12 +6,16 @@ category: technology
+When we speak of stateless (and [stateful](https://glossary.cncf.io/stateful_apps/)) apps, state refers to any data the app needs to store to function as designed. Stateless apps, as the name suggests, neither read nor store state information. You already use stateless applications on a regular basis. When you use a search engine to pull up a query about a question you have, you’re using a stateless application. Even if you’re interrupted during the process, you can come straight back and ask the same question, or a different question, because all memory of the previous transaction is gone between takes. 
+
+
 
 ## Problem it addresses
 
-Stateless applications tackle the problem of resiliency, because different pods across a [cluster](/cluster/) can work independently, with multiple requests coming to them at the same time. If there’s a problem, you can easily restart the application, and it will return to its initial state with little or no downtime. As such, the benefits of stateless applications include resiliency, elasticity, and high availability. However, most applications we use today are at least partly [stateful](/stateful_apps/), as they store things like preferences and settings to improve the user experience.
+When it comes to stateful applications it has to store the details of every transaction that happened in the previous step. So the need for extensive memory in the form of databases and servers can make stateful applications  slower, heavier, harder to manage. 
+
+Here in stateless applications, every transaction is fresh as the moment it was born, this eliminates the need for large memory overheads. If there’s a problem, you can easily restart the application, and it will return to its initial state with little or no downtime.
 
 ## How it helps
 
-Boiling everything down, in a Stateless Application the only thing your cluster is responsible for is the code, and other static content, being hosted on it. That’s it, no changing databases, no writes and no left over files when the pod is deleted. Stateless [containers](/container/) are easier to deploy, and you don’t need to worry about saving container data on persistent storage volumes. You also don't have to worry about backing up the data.
+Stateless Protocols/applications works better at the time of crash/accidentally if your application is closed because there is no state that must be restored, a failed server can simply restart after a crash. hence you don’t have to worry about backing up the data if an application/server crashes. Companies can easily add or remove new instances of the application whenever necessary.
```

Comment:
Because the last paragraph in the previous section actually belongs here, it is a little repetitive. I'd move the previous paragraph here and add the "add or remove instances."  

> For stateless applications, every transaction is a fresh start. Without memory overheads, the application can quickly restart whenever a problem occurs. Because it will always return to its initial state, there is no need to pull data from the server, minimizing downtime. 
> 
> Another benefit is that companies can easily scale the app up and down by adding or removing new instances without worrying about syncing or backing up data. All this makes stateless applications ideal for the web, where connections are often disrupted, servers are [distributed](https://glossary.cncf.io/distributed_systems/) across the globe, and speed is of the essence.

Reply:
I think your suggestion is close to the exact wording we should use.